libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSession.kt

```diff
@@ -42,20 +43,33 @@ internal class MapboxDirectionsSession(
     @RoutesExtra.RoutesUpdateReason
     private var routesUpdateReason: String = RoutesExtra.ROUTES_UPDATE_REASON_CLEAN_UP
 
-    override var initialLegIndex = 0
+    override var initialLegIndex = DEFAULT_INITIAL_LEG_INDEX
         private set
 
+    private companion object {
+        private const val DEFAULT_INITIAL_LEG_INDEX = 0
+
+        private fun SetRoutes.initialLegIndex(): Int =
```

Comment:
Shouldn't this be in `SetRoutesEx`?

Reply:
it's needed only in the particular class, so we're good to keep it here